infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -222,6 +236,21 @@ private void initWriterAndBufferMgr() {
                 metadataManager.getReplicationMetadata(session).getLastLogEntryBatchProcessed(), this);
     }
 
+    private void insertQInRegistryTable() {
+        TableRegistry tableRegistry = runtime.getTableRegistry();
+        String replicatedQName = REPLICATED_QUEUE_NAME_PREFIX + session.getSourceClusterId();
+        if (!tableRegistry.getRegistryTable().containsKey(replicatedQName)) {
+            try {
+                tableRegistry.registerTable(CORFU_SYSTEM_NAMESPACE, replicatedQName, Queue.CorfuGuidMsg.class,
+                        Queue.RoutingTableEntryMsg.class, Queue.CorfuQueueMetadataMsg.class,
+                        TableOptions.fromProtoSchema(Queue.RoutingTableEntryMsg.class));
```

Comment:
we can't use `fromProtoSchema` since the protobuf won't have the necessary stream tags that @souravgarg 's code might look for - can we instead construct `TableOptions` manually to fill in the necessary stream tag in this case just `lrq_recv_<client_name>`?